core/src/main/resources/error/error-classes.json

```diff
@@ -46,9 +70,20 @@
     "message" : [ "Failed to rename %s to %s as destination already exists" ],
     "sqlState" : "22023"
   },
+  "FAILED_RENAME_TEMP_FILE" : {
+    "message" : [ "Failed to rename temp file %s to %s as rename returned false" ],
+    "sqlState" : "42000"
```

Comment:
This isn't necessarily a syntax error. I don't have a good idea for which SQLSTATE this should be though. I'd leave this empty.